proto/cosmos/counter/v1/tx.proto

```diff
@@ -17,6 +18,7 @@ service Msg {
 
 // MsgIncreaseCounter defines a count Msg service counter.
 message MsgIncreaseCounter {
+  option (amino.name)           = "cosmos-sdk/increase_counter"; // TODO: remove amino
```

Comment:
aux builder test fails without this but its not required anymore, ill look into the test to see if we can remove this here